app/views/govuk_component/lead_paragraph.raw.html.erb

```diff
@@ -0,0 +1,13 @@
+<%
+  text ||= ""
+  def nbsp_between_last_two_words(text)
+    return text if text.nil?
```

Comment:
This first line will never happen so can be omitted – we set it to an empty string in the line above

Reply:
Fixed this. (Hopefully also squashed the linting problem.)